frontend/src/pages/Search.vue

```diff
@@ -83,7 +80,7 @@ export default {
   resources: {
     search: {
       cache: 'Search',
-      url: 'gameplan.api.search',
+      url: 'frappe_search.core.search',
```

Comment:
`frappe_search.api.search` looks better and is more semantic? minor naming nitpick